server/src/main/scala/actors/DrtSystemInterface.scala

```diff
@@ -339,7 +347,7 @@ trait DrtSystemInterface extends UserRoleProviderLike {
   }
 
   def createLiveChromaFlightFeed(feedType: ChromaFeedType): ChromaLiveFeed = {
-    ChromaLiveFeed(new ChromaFetcher[ChromaLiveFlight](feedType, ChromaFlightMarshallers.live) with ProdSendAndReceive)
+    ChromaLiveFeed(new ChromaFetcher[ChromaLiveFlight](feedType, ChromaFlightMarshallers.live) with MockedChromaSendReceive)
```

Comment:
Should this be `MockedChromaSendReceive`?

Reply:
oops! nice catch.